jsonschema-validation.xml

```diff
@@ -634,12 +620,12 @@
 
             <section title="anyOf">
                 <t>
-                    This keyword's value MUST be an array. This array MUST have at least one
-                    element.
+                    This keyword's value MUST be an array.
+                    This array MUST have at least one element.
                 </t>
                 <t>
-                    Elements of the array MUST be objects. Each object MUST be a valid JSON
-                    Schema.
+                    Elements of the array MUST be objects.
```

Comment:
I think we just want to say

Each element of the array MUST be a valid JSON Schema.

in order to allow boolean schemas as well.